README.rst

```diff
@@ -1,6 +1,11 @@
 codejail plugin for `Tutor <https://docs.tutor.overhang.io>`__
 ===================================================================================
 
+The repository Tutor contrib codejail, is a codejail plugin to manages execution of
+untrusted code in secure sandboxes. It is designed primarily for Python execution, but
+can be used for other languages as well. Security is enforced with AppArmor.
```

Comment:
```suggestion
Tutor plugin that enables execution of untrusted code in secure sandboxes using an external [service](https://github.com/eduNEXT/codejailservice) based on the [codejail](https://github.com/openedx/codejail) library.
```
Most of what you described here is part of codejail itself, and this is just a tutor plugin that helps with tutor + codejail.